composer.json

```diff
@@ -15,7 +15,7 @@
 		"10up/phpcs-composer": "dev-master",
 		"phpunit/phpunit": "9.4.4",
 		"yoast/phpunit-polyfills": "1.x-dev",
-		"automattic/vipwpcs": "dev-develop"
+		"automattic/vipwpcs": "^2.3"
```

Comment:
Curious why we include this here? Above we include the `10up/phpcs-composer` dependency which will include the `automattic/vipwpcs` dependency itself.

Reply:
That's a good point. It was added in PR #212, I removed it in [454a0b6](https://github.com/10up/restricted-site-access/pull/268/commits/454a0b67cec367597a63f469bd05304254ef7ed9)